gpflow/optimizers/scipy.py

```diff
@@ -7,16 +7,15 @@
 
 __all__ = ["Scipy"]
 
-Variables = Iterable[tf.Variable]
```

Comment:
not backwards-compatible ... perhaps deprecate instead?

Reply:
I'd argue it's not part of the "interface" as it's not included in `__all__` ... how would you deprecate a type ?

Reply:
`__all__` doesn't define the public API - it defines how `from foo import *` works

oh, I don't mean "officially", just for our sake, with e.g. `# deprecated`

Reply:
for our sake it had only been used in this module though

Reply:
we can't know that - Joe Bloggs might have imported in when using 2.0.1

Reply:
urgh. alright

Reply:
lol